package.json

```diff
@@ -32,19 +32,19 @@
   "author": "Art.sy Inc",
   "license": "MIT",
   "dependencies": {
-    "@babel/cli": "7.0.0-beta.41",
-    "@babel/core": "7.0.0-beta.41",
-    "@babel/node": "^7.0.0-beta.41",
-    "@babel/preset-env": "7.0.0-beta.41",
-    "@babel/preset-es2015": "7.0.0-beta.41",
-    "@babel/preset-stage-3": "^7.0.0-beta.41",
-    "@babel/preset-typescript": "7.0.0-beta.41",
-    "@babel/register": "7.0.0-beta.41",
+    "@babel/cli": "7.0.0-beta.46",
+    "@babel/core": "7.0.0-beta.46",
+    "@babel/node": "^7.0.0-beta.46",
+    "@babel/preset-env": "7.0.0-beta.46",
+    "@babel/preset-es2015": "7.0.0-beta.46",
+    "@babel/preset-stage-3": "^7.0.0-beta.46",
+    "@babel/preset-typescript": "7.0.0-beta.46",
+    "@babel/register": "7.0.0-beta.46",
     "@heroku/foreman": "^2.0.2",
     "accounting": "^0.4.1",
-    "apollo-link": "^1.0.0",
-    "apollo-link-context": "^1.0.0",
-    "apollo-link-http": "^1.1.0",
+    "apollo-link": "1.2.1",
+    "apollo-link-context": "^1.0.8",
+    "apollo-link-http": "^1.5.4",
     "artsy-morgan": "git://github.com/artsy/artsy-morgan.git",
```

Comment:
The specific apollo version is because of https://github.com/apollographql/graphql-tools/pull/765